www/docs/topic/image.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+include_once '../../includes/easyparliament/init.php';
+
+$topics = new \MySociety\TheyWorkForYou\Topics();
+$slug = get_http_var('id');
+if ($slug) {
+    $topic = $topics->getTopic($slug);
+}
+
+if (!$slug || !isset($topic) || $topic === NULL) {
+    header('HTTP/1.0 404 Not Found');
+    exit();
+}
+
+$path = $topic->image_path();
+
+if (!$path || !file_exists($path) || !is_file($path) || !is_readable($path)) {
```

Comment:
Does is_readable not do the file_exists check and the string being blank for you?